packages/components-native/src/Divider/DividerVertical.style.ts

```diff
@@ -0,0 +1,27 @@
+import { StyleSheet } from "react-native";
+import { tokens } from "../utils/design";
+
+export const verticalStyles = StyleSheet.create({
+  base: {
+    margin: 0,
+    height: "100%",
+    width: "1px",
+    borderRightWidth: tokens["border-base"],
+    borderRightColor: tokens["color-border"],
+  },
+  large: {
+    borderRightWidth: tokens["border-thick"],
+    opacity: 0.875,
+  },
+  larger: {
+    borderRightWidth: tokens["border-thicker"],
+    opacity: 0.625,
+  },
+  largest: {
+    borderRightWidth: tokens["space-small"],
```

Comment:
I know this is just a copy over from the existing styles but I think it should also be updated to no longer user space as well as the original

Reply:
Would suggest using `border-thickest` here ([source](https://atlantis.getjobber.com/?path=/docs/design-borders--docs#border-widths))